app/models/container_service.rb

```diff
@@ -12,6 +12,7 @@ class ContainerService < ActiveRecord::Base
   has_many :labels, -> { where(:section => "labels") }, :class_name => "CustomAttribute", :as => :resource, :dependent => :destroy
   has_many :selector_parts, -> { where(:section => "selectors") }, :class_name => "CustomAttribute", :as => :resource, :dependent => :destroy
   has_many :container_nodes, -> { distinct }, :through => :container_groups
+  belongs_to :container_image_registry
```

Comment:
Just wondering how openshift works exactly: Theres a service that caters to an image registry? 
@moolitayer @abonas